moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -106,6 +106,20 @@ const ProjectSummaryKnackDataTrackerSync = ({
           (fieldMap[process.env[regExResult[0]]] = regExResult[1].toLowerCase())
       );
 
+    console.log(project);
```

Comment:
Should we delete this console log? I assume it was just needed for local dev/testing.

Reply:
You bet, this console.log() has been removed. 🙏